Updated Abstract (#12)
tlswg/tls-key-update: draft-ietf-tls-extended-key-update.md

     date: November 2023

--- abstract
The Transport Layer Security (TLS) 1.3 specification provides forward secrecy

> by utilizing a Diffie-Hellman key exchange during the initial handshake. Forward

Reviewer: I suggest to replace "Diffie-Hellman" with "ephemeral key exchange"

> by utilizing a Diffie-Hellman key exchange during the initial handshake. Forward

Author: ```suggestion
by utilizing an ephemeral key exchange during the initial handshake. Forward
```

secrecy ensures that even if an attacker later obtains a party's long-term private
key, past encrypted sessions cannot be decrypted. This protects against adversaries
who record encrypted conversations in the hope of decrypting them later.

TLS 1.3 also includes a Key Update mechanism, allowing cryptographic keys to be
refreshed during an ongoing session. However, this update does not establish new
forward-secret key material. While this is generally not an issue for short-lived
sessions, it can pose a security risk for long-lived connections, such as those in
industrial IoT or telecommunication networks, where an attacker could compromise
application traffic secrets after the initial handshake.

Earlier versions of TLS supported renegotiation, a mechanism that allowed peers to

> establish new cryptographic parameters within an existing session. However, due to

Reviewer 2: Maybe to add, session renegotiation did not only allow to establish new cryptographic parameter for the sesssion, it also allowed to utilized an updated long term credential (certificate). This adresses cases, were the initially used long term credential may have reached the validity end or may have been revoked in the meantime. 

Proposal to change the sentence to:
Earlier versions of TLS supported session renegotiation, a mechanism that allowed peers to
establish new cryptographic parameters within an existing session including the potential update of the initially utilized long term keys (certificates) with renewed credentials.
